N.au3

```diff
@@ -40,13 +40,13 @@ endfunc
 func NMain($ep)
 	if NIsMain() then
 		DllCallAddress('none', $__nPfn_PrepMain)
-		Call($ep)
+		Call(IsFunc($ep) ? FuncName($ep) : $ep)
 	else
 		; Retrieve function name and local.
 		local $s = DllStructCreate('char[64];')
 		local $l = DllCallAddress('idispatch', $__nPfn_PrepSub, 'ptr', DllStructGetPtr($s, 1))[0]
 		local $fn = DllStructGetData($s, 1)
-		Call($fn, $l)
+		Call(IsFunc($fn) ? FuncName($fn) : $fn, $l)
```

Comment:
The **Call** function accepts both function handle and name.
But in this case, it always gets a string from `$fn`.